src/fides/api/ops/api/v1/endpoints/policy_endpoints.py

```diff
@@ -148,6 +148,75 @@ def create_or_update_policies(
     )
 
 
+def get_rule_or_error(
+    db: Session, policy_key: FidesOpsKey, rule_key: FidesOpsKey
+) -> Rule:
+    """
+    Helper method to load Rule or throw a 404 if it can't be found.
+    Also throws a 404 if a `Policy` with the given key can't be found.
+    """
+    logger.info("Finding rule with key '{}'", rule_key)
+    policy = get_policy_or_error(db, policy_key)
+    rule = Rule.filter(
+        db=db,
+        conditions=((Rule.policy_id == policy.id) & (Rule.key == rule_key)),
+    ).first()
+    if not rule:
+        raise HTTPException(
+            status_code=HTTP_404_NOT_FOUND,
+            detail=f"No Rule found for key {rule_key}.",
+        )
+
+    return rule
+
+
+@router.get(
+    urls.RULE_LIST,
+    status_code=HTTP_200_OK,
+    response_model=Page[schemas.RuleResponseWithTargets],
+    dependencies=[Security(verify_oauth_client, scopes=[scope_registry.RULE_READ])],
+)
+def get_rule_list(
+    *,
+    db: Session = Depends(deps.get_db),
+    policy_key: FidesOpsKey,
+    params: Params = Depends(),
+) -> AbstractPage[Rule]:
+    """
+    Return a paginated list of all `Rule` records associated with the given `Policy`.
+    Throws a 404 if the given `Policy` can't be found.
+    """
+    logger.info(
+        "Finding all rules for policy {} with pagination params '{}'",
+        policy_key,
+        params,
+    )
+    policy = get_policy_or_error(db, policy_key)
+    rules = Rule.filter(
+        db=db,
+        conditions=(Rule.policy_id == policy.id),
+    ).order_by(Rule.created_at.desc())
+    return paginate(rules, params=params)
+
+
+@router.get(
+    urls.RULE_DETAIL,
+    status_code=HTTP_200_OK,
+    response_model=schemas.RuleResponseWithTargets,
+    dependencies=[Security(verify_oauth_client, scopes=[scope_registry.RULE_READ])],
+)
+def get_rule(
+    *,
+    policy_key: FidesOpsKey,
+    rule_key: FidesOpsKey,
+    db: Session = Depends(deps.get_db),
+) -> schemas.Rule:
```

Comment:
I'd update return type to the Rule sqlalchemy model and then hopefully you cam remove your type ignore below.